docs/release-notes/breaking-changes.md

```diff
@@ -21,6 +21,10 @@ Breaking changes can impact your Elastic applications, potentially disrupting no
 % **Action**<br> Steps for mitigating deprecation impact.
 % ::::
 
+## 9.2.1 [9-2-1]
+
+_No breaking changes_ 
+
```

Comment:
I think all the `9.2.1` should be commented out? I am a bit confused with the release notes PR as a whole. Since the purpose of `run-patch-release` here was to bump up versions and `9.2.1` is not going to be released currently but in future I don't think it makes sense to generate these PR. And if they are indeed generated, they should be commented out completely so as to not publish an unreleased version.